templates/app/client/components/auth(auth)/auth.service.js

```diff
@@ -148,7 +148,7 @@ export function AuthService($location, $http, $cookies, $q, appConfig, Util, Use
      * @return {Bool}
      */
     isLoggedInSync() {
-      return currentUser.hasOwnProperty('role');
+      return currentUser.hasOwnProperty('role') && currentUser.role !== '';
```

Comment:
This is correct. Fixes the issue for me. Also this should be duplicated for the async version of `isLoggedIn`